### PR TITLE
ci: refactor main.dox updates

### PR DESCRIPTION
**Pull request by coryan**
I think this makes the work to generate new libraries easier.

- - -
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/9697)

### CODE REVIEW

File: ci/cloudbuild/builds/checkers.sh [hunk 1]
```diff
@@ -191,5 +191,38 @@ time {
   done
 }
 
+printf "%-30s" "Updated generated doxygen:" >&2
```
Comment (dbolduc) [inline ci/cloudbuild/builds/checkers.sh:194]: nit: s/Updated/Updating/

**Review by dbolduc — APPROVED** [review]

File: ci/cloudbuild/builds/checkers.sh (hunk 1 above)
Comment (coryan) [reply ci/cloudbuild/builds/checkers.sh]: Fixed.

File: ci/cloudbuild/builds/checkers.sh [hunk 2]
```diff
@@ -191,5 +191,38 @@ time {
   done
 }
 
+printf "%-30s" "Updating generated doxygen:" >&2
```
[the remark this answers is not in the document]
Comment (devbww) [reply ci/cloudbuild/builds/checkers.sh:194]: Indeed, ...
```
======================================                                          
|   Starting local build: checkers   |                                          
======================================                                          
2022-08-17T17:14:20Z (+1s): Checking for a valid git environment...             
2022-08-17T17:14:20Z (+1s): Inside a git repository                             
Running typos:                ... 1.886 seconds                                 
Running check-include-guards: ... 0.272 seconds                                 
Running Abseil header fixes:  ... 1.225 seconds                                 
Running include fixes:        ... 1.233 seconds                                 
Running whitespace fixes:     ... 1.470 seconds                                 
Running buildifier:           ... 0.134 seconds                                 
Running black:                ... 0.229 seconds                                 
Running shfmt:                ... 0.042 seconds                                 
Running shellcheck:           ... 0.195 seconds                                 
Running clang-format:         ... 6.446 seconds                                 
Running cmake-format:         ... 2.515 seconds                                 
Running markdown generators:  ... 5.634 seconds                                 
Updating generated doxygen:   <NEXT-COMMAND-OUTPUT>
```
And, if you revisit, perhaps `Updating ...` to `Running ...` would look nicer.

[the remark this answers is not in the document]
Comment (coryan) [reply ci/cloudbuild/builds/checkers.sh:194]: I did not disable the auto merge in time.  I will send a follow up PR asap.

**Pull request merged**